Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.c

```diff
@@ -51,6 +51,7 @@ static size_t pgm_metadata_index = 0;
 uint16_t pas_probabilistic_guard_malloc_random;
 uint16_t pas_probabilistic_guard_malloc_counter = 0;
 
+bool pas_probabilistic_guard_malloc_has_been_used = false;
```

Comment:
You should make it `static` variable in this file and expose a function returning this information.